db/migrate/20160825213523_change_load_balancer_listener_port_to_port_range.rb

```diff
@@ -0,0 +1,55 @@
+class ChangeLoadBalancerListenerPortToPortRange < ActiveRecord::Migration[5.0]
+  def change
+    reversible do |dir|
+      dir.up do
+        # convert load_balancer_port and instance_port into ranges, preserving
+        # existing values
+        execute <<-SQL
+          ALTER TABLE load_balancer_listeners
+            ALTER COLUMN load_balancer_port SET DATA TYPE int4range
```

Comment:
When you said to arel, I didn't realize you meant the SQL manipulation layer in ActiveRecord; instead I was trying to figure out how to write raw arel code.  Whoops!

What you wrote above totally works. The only difference I see is that each record is updated one at a time (i.e. multiple sql statements are run for each update). This might be slow (doesn't matter for a one-off migration), and it might have race conditions with other updates.

However given this is a change that we don't expect anyone to be using yet, I highly doubt someone will run the migration and also be manipulating loadbalancers at the same time. I'm happy to use this format above instead.